test/fixtures/keys/selfsigned-no-keycertsign/https_renew_cert.sh

```diff
@@ -1,6 +1,12 @@
-#!/bin/bash
-openssl genrsa -out rsa.pem 2048
-openssl rsa -in rsa.pem -out key.pem
-openssl req -sha256 -new -key key.pem -out csr.pem -subj "/CN=localhost"
-openssl x509 -req -extfile cert.conf -extensions v3_req -days 365 -in csr.pem -signkey key.pem -out cert.pem
+#!/usr/bin/env bash
+set -euo pipefail
+shopt -s inherit_errexit
+
+cd -- "$(dirname -- "${BASH_SOURCE[0]}")"
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2164
```suggestion
cd "$( dirname "$0" )" || exit
```

Reply:
That should not be necessary with `set -e` in this case. The reference you provided says that ShellCheck will not flag this when `set -e` is found.